pandas/core/series.py

```diff
@@ -1885,10 +1885,110 @@ def update(self, other):
     # ----------------------------------------------------------------------
     # Reindexing, sorting
 
-    @Appender(generic._shared_docs['sort_values'] % _shared_doc_kwargs)
+    #@Appender(generic._shared_docs['sort_values'] % _shared_doc_kwargs)
```

Comment:
We have removed the Apender because it was not generic. It was made for DataFrame.